extensions/ql-vscode/src/vscode-tests/run-integration-tests.ts

```diff
@@ -78,10 +78,11 @@ async function main() {
     const extensionTestsEnv: Record<string, string> = {};
     if (dirs.includes(TestDir.CliIntegration)) {
       console.log('Installing required extensions');
-      const cliPath = resolveCliPathFromVSCodeExecutablePath(vscodeExecutablePath);
+      const [cli, ...args] = resolveCliArgsFromVSCodeExecutablePath(vscodeExecutablePath);
```

Comment:
The comment on this function was "Usually you will want `resolveCliArgsFromVSCodeExecutablePath` instead.", so it's probably better to use that function.